e2e-tests/src/utils.rs

```diff
@@ -628,7 +624,11 @@ pub async fn new_account(config: &TestConfig) -> eyre::Result<PrivateKeySigner>
     tx.set_to(new_signer.address());
     tx.set_value(U256::from(10 * ETH_TO_WEI));
     let send = provider.send_transaction(tx).await?;
-    let _ = send.get_receipt().await?;
+    let _ = send.with_required_confirmations(1).get_receipt().await?;
```

Comment:
get_receipt should be enough. Why wait one more block to confirm? Do you see something not expected?